scan

```diff
@@ -94,6 +98,8 @@ while [[ $# > 0 ]]; do
 
   -xo|--driver-options) shift; DRIVER_OPTION=$1;;
 
+  --brightness) BRIGHTNESS=1 ;;
```

Comment:
I see this hard-codes argument `20x10` in ImageMagick. I would prefer we mirror the ImageMagick argument name "--brightness-contrast" (really `--brightness-contrast-sw` as per comment below) and then accept the brightness-contrast argument from ImageMagick so that we are exposing this capability in a configurable way.

People can always use the config file if a particular value works for them all the time.